examples/get_started/torch-loader.py

```diff
@@ -80,7 +80,10 @@ def forward(self, x):
     # Train the model
     for epoch in range(NUM_EPOCHS):
         with tqdm(
-            train_loader, desc=f"epoch {epoch + 1}/{NUM_EPOCHS}", unit="batch"
+            train_loader,
+            desc=f"epoch {epoch + 1}/{NUM_EPOCHS}",
+            unit="batch",
+            leave=False,
```

Comment:
I find this useful. Besides, this is an example. 

I'll remove this if you don't mind.